### DISCUSSION TITLE
Logs not generated for Custom Timetable with CeleryExecutor

### DISCUSSION THREAD

**Discussion by jyotsa09** [apache/airflow#23218] [General]
### Apache Airflow version

2.2.4

### What happened

Logs not generated for Custom Timetable with CeleryExecutor

### What you think should happen instead

Dag should be successful

### How to reproduce

1. Create a custom timetable and dag that is using the custom timetable
2. Trigger the dag

### Operating System

debian

### Deployment

Astronomer

### Deployment details

astro deploy

### Anything else

Custom Timetable - https://gist.github.com/jyotsa09/afc633c3374ae98405cc4a8873ddf29c
Dag - https://gist.github.com/jyotsa09/b65caf1d9c375055f886846dc6d40db6

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/apache/airflow/blob/main/CODE_OF_CONDUCT.md)

**Comment by jyotsa09** [comment]
Got this issue while testing - https://github.com/apache/airflow/issues/20475

**Comment by uranusjr** [comment]
The custom timetable sems very similar to the built-in `CronDataIntervalTimetable`. Are they the same? If not, can you summarise their differences? I am trying to determine whether this is caused by

* The fact that this timetable is not built-in,
* Something’s wrong in the timetable implementation,
* Something’s wrong with the executor not related to timetables.

Also, does everything work as expected if you change the DAG to use the built-in `from airflow.timetables.interval import CronDataIntervalTimetable` instead?

**Comment by potiuk** [comment]
Converted it to a discussion until we gather more information about it from @jyotsa09
